justfile

```diff
@@ -46,11 +46,11 @@ validate-modified-chains REF:
 
 # Run validation checks for chains with a name or chain ID matching the supplied regex, example: just validate 10
 validate CHAIN_ID:
-	TEST_DIRECTORY=./validation go run gotest.tools/gotestsum@latest --format testname -- -run=TestValidation/{{CHAIN_ID}} -count=1
+	TEST_DIRECTORY=./validation go run gotest.tools/gotestsum@latest --format testname -- -run='TestValidation/.+\({{CHAIN_ID}}\)$' -count=1
 
 # Run genesis validation (this is separated from other validation checks, because it is not a part of drift detection)
 validate-genesis CHAIN_ID:
-	TEST_DIRECTORY=./validation/genesis go run gotest.tools/gotestsum@latest --format testname -- -run=TestGenesisPredeploys/{{CHAIN_ID}} -v
+	TEST_DIRECTORY=./validation/genesis go run gotest.tools/gotestsum@latest --format testname -- -run='TestGenesisPredeploys/.+\({{CHAIN_ID}}\)$' -v -count=1
```

Comment:
Let's get these changes into `main` (along with the accompanying changes in the circleci config).